agent/agent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/TracerInstaller.java

```diff
@@ -40,21 +38,10 @@ public static void installAgentTracer() {
         OpenTelemetry.setPropagators(
                 DefaultContextPropagators.builder().addTextMapPropagator(AiHttpTraceContext.getInstance()).build());
 
-        double samplingPercentage = Global.getSamplingPercentage();
-        if (samplingPercentage != 1) {
-            OpenTelemetrySdk.getTracerManagement().updateActiveTraceConfig(
-                    TraceConfig.getDefault().toBuilder()
-                            .setSampler(new TraceIdBasedSampler(samplingPercentage))
-                            .build());
-        } else {
-            // OpenTelemetry default sampling is "parent based", which means don't sample if remote traceparent sampled flag was not set,
```

Comment:
~~should we keep the else block.. this is a specific fix for azure function sending false sampler flag...~~